test/e2e/specs/find-dojo.spec.js

```diff
@@ -13,10 +13,10 @@ describe('Find Dojo page', () => {
     DojoPage.addressSearchInput.setValue('CHQ');
     DojoPage.addressSearchButton.click();
     DojoPage.showDojoListCount.waitForVisible();
-    expect(DojoPage.showDojoListCount.getText()).to.have.string('Showing 5 Dojos');
+    expect(DojoPage.showDojoListCount.getText()).to.have.string('Showing 1 to 6 of 8 Dojos');
```

Comment:
Same as for dates, get the number of dojos instead or anybody will have to update those numbers regularly